kiali/README.md

```diff
@@ -40,22 +40,40 @@ An alternative can be a parallel installation of the upstream chart offering all
 
 1. Run the Helm upgrade command, which installs the chart if not present yet.
     ```bash
-    helm upgrade --install --create-namespace -n ${KYMA_KIALI_NS} ${HELM_RELEASE_NAME} kiali/kiali-operator --set cr.spec.auth.strategy=anonymous -f https://raw.githubusercontent.com/kyma-project/examples/main/kiali/values.yaml
+    helm upgrade --install --create-namespace -n $KYMA_KIALI_NS $HELM_RELEASE_NAME kiali/kiali-operator -f https://raw.githubusercontent.com/kyma-project/examples/main/kiali/values.yaml
     ```
 
-You can either use the `[values.yaml](./values.yaml)` provided in this `kiali` folder, which contains customized settings deviating from the default settings, or create your own `values.yaml` file.
+You can either use the [`values.yaml`](./values.yaml) provided in this `kiali` folder, which contains customized settings deviating from the default settings, or create your own `values.yaml` file.
 
 ### Verify the installation
 
-1. You should see the `kiali-operator` and `kiali` Pod coming up in the Namespace. All Pods must eventually be in a "Running" state.
-1. Browse the Kiali dashboard. The following command exposes the dashboard on `http://localhost:20001`:
-   ```bash
-   kubectl -n ${KYMA_KIALI_NS} port-forward svc/kiali-server 20001
-   ```
+Check that the `kiali-operator` and `kiali-server` Pods have been created in the Namespace and are in the `Running` state:
+```bash
+kubectl wait --for=condition=Ready pod $(kubectl -n $KYMA_KIALI_NS get pods --no-headers -o custom-columns=":metadata.name" | grep kiali) -n $KYMA_KIALI_NS
```

Comment:
```suggestion
kubectl -n $KYMA_KIALI_NS rollout status deploy $HELM_RELEASE_NAME-kiali-operator && kubectl -n $KYMA_KIALI_NS rollout status deploy kiali-server
```

I forgot that if the `kiali-server` Pod has not been created yet, the condition would be met anyway, because it would only test for condition the existing Pods, so this command will not do...

You could go with expanding my suggestion to this: 
```bash
kubectl wait --for=condition=Ready pod $(kubectl -n $KYMA_KIALI_NS \
get pods --no-headers -o custom-columns=":metadata.name" | grep kiali-operator) -n $KYMA_KIALI_NS \
&& kubectl wait --for=condition=Ready pod $(kubectl -n $KYMA_KIALI_NS \
get pods --no-headers -o custom-columns=":metadata.name" | grep kiali-server) -n $KYMA_KIALI_NS
```
If `kiali-server` has not been created yet, this results in: 
```
pod/{HELM-RELEASE_NAME}-kiali-operator-6db8d4777f-bm9d8 condition met
error: resource(s) were provided, but no name was specified
```

It's jut not the prettiest... 


**But luckily, I learnt a cool new command in the meantime!** 

`kubectl rollout status deploy` - and it works!

<img src="https://user-images.githubusercontent.com/53601578/194089722-0ebb0eb4-8aa8-4e4e-aa7d-50ff5bac932b.png" width="300"> ... ⌛ ... <img src="https://user-images.githubusercontent.com/53601578/194089789-a84f010a-0c8e-4d05-b7ae-26b19d063b95.png" width="300"> 

So I'm adding a new suggestion here 😊